modular_skyrat/modules/reagent_forging/code/forge.dm

```diff
@@ -462,6 +470,35 @@
 		COOLDOWN_START(spawned_glass, remaining_heat, glassblowing_amount)
 		return TRUE
 
+	if(istype(attacking_item, /obj/item/food))
+		var/obj/item/food/thing_to_cook = attacking_item
+		if(in_use)
+			to_chat(user, span_warning("You cannot do multiple things at the same time!"))
+			return
+		in_use = TRUE
+		if(forge_temperature < MIN_FORGE_TEMP)
+			fail_message(user, "The [src] is not hot enough to start cooking [thing_to_cook]!")
+			return
+		var/user_input = show_radial_menu(user, src, radial_options)
+		var/obj/item_to_spawn
+		if(!user_input)
+			fail_message(user, "No choice made")
+			return
+		balloon_alert_to_viewers("cooking...")
+		if(!do_after(user, 10 SECONDS, target = src))
+			fail_message(user, "You stop trying to cook [thing_to_cook]!")
+			return
+		switch(user_input)
+			if("oven")
+				var/datum/component/bakeable/item_bakeable_component = thing_to_cook.GetComponent(/datum/component/bakeable)
+				item_to_spawn = item_bakeable_component.bake_result ? item_bakeable_component.bake_result : /obj/item/food/badrecipe
+			if("microwave")
+				item_to_spawn = thing_to_cook.microwaved_type ? thing_to_cook.microwaved_type : /obj/item/food/badrecipe
+		qdel(thing_to_cook)
+		new item_to_spawn(get_turf(src))
+		in_use = FALSE
+		return TRUE
```

Comment:
First of all, if you're going to have a proc that's meant to return true at the end, then you're going to want to explicitly return false when it fails. I know the rest of the proc wasn't doing it, but I also forgot to mention that back when that code was first added.

Secondly, let it breathe, space shit out, it really helps readability. We're not running out of space for whitespaces, so don't be shy and use 'em.
```suggestion
	if(istype(attacking_item, /obj/item/food))
		var/obj/item/food/thing_to_cook = attacking_item

		if(in_use)
			to_chat(user, span_warning("You cannot do multiple things at the same time!"))
			return FALSE

		in_use = TRUE
	
		if(forge_temperature < MIN_FORGE_TEMP)
			fail_message(user, "The [src] is not hot enough to start cooking [thing_to_cook]!")
			return FALSE

		var/user_input = show_radial_menu(user, src, radial_options)
		var/obj/item_to_spawn

		if(!user_input)
			fail_message(user, "No choice made")
			return FALSE

		balloon_alert_to_viewers("cooking...")

		if(!do_after(user, 10 SECONDS, target = src))
			fail_message(user, "You stop trying to cook [thing_to_cook]!")
			return FALSE

		switch(user_input)
			if("oven")
				var/datum/component/bakeable/item_bakeable_component = thing_to_cook.GetComponent(/datum/component/bakeable)
				item_to_spawn = item_bakeable_component.bake_result ? item_bakeable_component.bake_result : /obj/item/food/badrecipe
			if("microwave")
				item_to_spawn = thing_to_cook.microwaved_type ? thing_to_cook.microwaved_type : /obj/item/food/badrecipe

		qdel(thing_to_cook)
		new item_to_spawn(get_turf(src))
		in_use = FALSE
		return TRUE
```